### PR TITLE
WIP: Add proper coafile

### PR DESCRIPTION
**Pull request by sils**
This isn't ready yet, @hemangsk there's some stuff to do here I think. This should've happened earlier ideally.

### CODE REVIEW

**Comment by hemangsk** [comment]
Hey @sils , do you think we can avoid lint check on the `bootstrap.min.css`

**Comment by hemangsk** [comment]
This is getting interesting 🤔

File: _includes/footer.html [hunk 1]
```diff
@@ -1,44 +1,35 @@
...
-        </ul>
-      </div>
-      <div class="col l4 offset-l2 s12">
-        <h5 class="white-text">Any Questions or Queries ?</h5>
-
-<div >
- <a href="https://gitter.im/viper-development/support" target="blank" class="mailbtn btn-large waves-effect grey-text text-lighten-4 activate-sidecar">Chat with us <span class=""><i class="mailicn fa fa-commenting" aria-hidden="true"></i>
-</span></a>
+  <div class="row">
+  <div class="col-md-8 s12">
```
Comment (sils) [inline _includes/footer.html:4]: this seems a functional change and not only codestyle, also why are you messing up indentation for this?

**Comment by hemangsk** [comment]
Opening a new PR due to increase in longitivity

**Pull request closed without merging**